CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 3.4)
-project(squirrel VERSION 3.2 LANGUAGES C CXX)
+project(squirrel
+  VERSION 3.2
+  DESCRIPTION "The Squirrel programming language"
+  HOMEPAGE_URL "http://squirrel-lang.org/"
```

Comment:
[`DESCRIPTION`](https://cmake.org/cmake/help/latest/variable/CMAKE_PROJECT_DESCRIPTION.html) is only supported since CMake 3.9 and [`HOMEPAGE_URL`](https://cmake.org/cmake/help/latest/variable/CMAKE_PROJECT_HOMEPAGE_URL.html) since 3.12.

I'm pretty sure that older CMakes will throw an error about that, but can't test right now.